src/components/product/Products.component.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
You can skip this line everywhere https://stackoverflow.com/questions/71725865/do-i-need-to-import-react-in-every-file